example_2.py

```diff
@@ -1,36 +1,26 @@
-import sys
-import config
-
 def main():
+    operations_count = 0
     ask_again = True
     while(ask_again):
-        if sys.version_info.major <= 2:
-            a = raw_input("Enter the numerator: ")
-            b = raw_input("Enter the denominator: ")
-        else:
-            a = input("Enter the numerator: ")
-            b = input("Enter the denominator: ")
+        a = input("Enter the numerator: ")
+        b = input("Enter the denominator: ")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The input method in Python 2 will read from standard input, evaluate and run the resulting string as python source code. This is similar, though in many ways worse, then using eval. On Python 2, use raw_input instead, input is safe in Python 3. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b322-input